litho-core/src/main/java/com/facebook/litho/CommonPropsHolder.java

```diff
@@ -640,6 +652,8 @@ public void copyInto(ComponentContext c, InternalNode node) {
     if ((mPrivateFlags & PFLAG_TEST_KEY_IS_SET) != 0L) {
       node.testKey(mTestKey);
     }
+    node.componentTag(mComponentTag);
```

Comment:
If `ComponentTag` is to be a Map it would be reasonable to add it only if the tag was set (i.e. by using a flag like `PFLAG_TEST_KEY_IS_SET`)